settings.gradle

```diff
@@ -21,6 +21,7 @@ includeWithFlags ':junit4',                              'java', 'publish', 'rel
 includeWithFlags ':junit5',                              'java', 'publish', 'relocate'
 includeWithFlags ':kafka',                               'java', 'publish', 'relocate'
 includeWithFlags ':kotlin',                              'java', 'publish', 'relocate', 'kotlin'
+includeWithFlags ':grpc-kotlin',                         'java', 'publish', 'relocate', 'kotlin'
```

Comment:
Could you sort this in alphabetical order?

Reply:
Done in [fa4d628](https://github.com/line/armeria/commit/fa4d628b0a113437da1d9ebb625332dad3b299be)!